CHANGELOG.md

```diff
@@ -1,4 +1,27 @@
 ## 2.13.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.4.1045** 
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Implemented support for ``enable_learned_cidrs_approval`` in **aviatrix_transit_gateway**, **aws_tgw_vpn_conn** and **aws_tgw_directconnect**
+2. Implemented a new parameter ``manage_transit_gateway_attachment`` to provide option of attaching/detaching transit gateway to/from AWS tgw outside of **aviatrix_aws_tgw**
+3. New resources for RBAC:
+  - **aviatrix_rbac_group**
+  - **aviatrix_rbac_group_access_account_attachment**
+  - **aviatrix_rbac_group_permission_attachment**
+  - **aviatrix_rbac_group_user_attachment**
+4. New resources
+  - **aviatrix_aws_tgw_transit_gateway_attachment**
+
+### Enhancements:
+1. Enhanced read back of ``attached_aviatrix_transit_gateway`` to cover the case where multiple transit gateways launched on the same vpc and one of them is attached to AWS tgw
+2. Removed ``account_name`` from **aviatrix_account_user** for RBAC implementation
+
+### Bug Fixes:
+1. Fixed issue where changes in vpc_name in **aviatrix_aws_tgw** results in ``subnets`` being mismatched in terraform causing issues
```

Comment:
1. Fixed issue where changes in the ``vpc_name`` in **aviatrix_aws_tgw** results in ``subnets`` being mismatched in the Terraform state